AsyncPoco.Tests/App.config

```diff
@@ -5,16 +5,16 @@
 		<add name="sqlserver" connectionString="Data Source=localhost;Initial Catalog=AsyncPocoTest;Integrated Security=True" providerName="System.Data.SqlClient" />
 		<add name="sqlserverce" connectionString="Data Source=petapoco.sdf" providerName="System.Data.SqlServerCe" />
 		<add name="postgresql" connectionString="Server=127.0.0.1;User id=postgres;password=password01;Database=postgres;" providerName="Npgsql" />
-
 	</connectionStrings>
 
 	<system.data>
 		<DbProviderFactories>
 			<clear />
 			<add name="Npgsql Data Provider" invariant="Npgsql" support="FF" description=".Net Framework Data Provider for Postgresql Server" type="Npgsql.NpgsqlFactory,Npgsql" />
-			<add name="MySQL Data Provider" invariant="MySql.Data.MySqlClient" description=".Net Framework Data Provider for MySQL" type="MySql.Data.MySqlClient.MySqlClientFactory,MySql.Data" />
-			<add name="Microsoft SQL Server Compact Data Provider 4.0" invariant="System.Data.SqlServerCe" description=".NET Framework Data Provider for Microsoft SQL Server Compact" type="System.Data.SqlServerCe.SqlCeProviderFactory, System.Data.SqlServerCe"/>
-		</DbProviderFactories>
+		  <remove invariant="MySql.Data.MySqlClient" />
```

Comment:
This "remove" element was automatically added when upgrading the MySQL reference. The reason it gets added is to avoid potential conflicts with developers that may have the assembly already referenced in their machine.config file.
Here's more info:  https://stackoverflow.com/questions/7354116/net-mysql-connector-conflicting-dbproviderfactories